ocpmodels/preprocessing/atoms_to_graphs.py

```diff
@@ -194,6 +215,15 @@ def convert(self, atoms: ase.Atoms, sid=None):
             data.fixed = fixed_idx
         if self.r_pbc:
             data.pbc = torch.tensor(atoms.pbc)
+        if self.r_data_keys is not None:
+            for (
+                data_key
+            ) in self.r_data_keys:  # if key is not present let it raise error
```

Comment:
One minor issue with raising an error here is that sometimes we'll have an inference dataset that doesn't contain ground truth values that we want to look up here. My suggestion would be to check for `if data_key in atoms.info` here, and let downstream code check for whether `data_key` ended up in `data`. Let me know if anything else is relying on an error being thrown here instead.

Reply:
Makes sense, added that check now. Though a downside is that a more obscure error may be thrown for cases where some `data_keys` were not present.